File: library_zap/src/main/java/com/dolin/zap/internal/IRecord.java
package com.dolin.zap.internal;

/**
 * @author #Suyghur.
 * Created on 2021/05/19
 */
public interface IRecord {

    void write(String msg);

    void asyncFlush();

    void asyncFlush(String path);

    void changeLogPath(String path);

    void expLogFile(String path, int limitSize);

    void release();
}

File: library_zap/src/main/java/com/dolin/zap/internal/IRecord.kt
package com.dolin.zap.internal

/**
 * @author #Suyghur.
 * Created on 4/7/21
 */
interface IRecord {

    fun write(msg: String)

    fun asyncFlush()

    fun expLogFile(path: String, partNum: Int)

    fun release()
}